automation/README.md

```diff
@@ -47,11 +47,13 @@ IntelliJ
 - set VM parameters `-Djsse.enableSNIExtension=false -Dheadless=false`
 - set Working dir to local dir
 - use classpath and SDK of the leonardoTests module
+- comment out `"-Xfatal-warnings"` on `Settings.scala`
```

Comment:
I'd rather fix the warnings than instruct people to comment this :)

Reply:
I think this change can be reverted....but the next comment is helpful...